lib/preview/constants.js

```diff
@@ -102,6 +102,14 @@ module.exports = {
     ],
     'PDF': [
       'application/pdf'
+    ],
+    'VIDEO': [
+      'video/mp4',
+      'video/mpeg',
+      'video/quicktime',
+      'video/x-m4v',
+      'video/x-ms-wmv',
+      'video/x-msvideo'
```

Comment:
A sampling of formats used to date in SuiteC, which we're reasonably confident that ffmpeg can handle out of the box.